### ISSUE TITLE
Studio 1.5 crashes when getting events 

### ISSUE THREAD

**Issue by FrancescoMondada** [aseba-community/aseba#483]
When running the following code on a Wireless Thymio or a Thymio connected by USB, under windows or macos, ASEBA Studio does not display the events and crashes. All is working fine under 1.4.
```
var speed=50
motor.left.target = speed
motor.right.target = speed

onevent prox
	when prox.ground.delta[0] < 400 do
		speed = speed+20
		motor.left.target = speed
		motor.right.target = speed
	end
	emit displayspeed speed
``` 
The problem seems to be at the reception and display of the events (here displayspeed )
Of course one should define an event called "displayspeed" with one argument.

**Comment by mbonani** [comment]
I confirm the problem on windows and macosx
If argument is different of zero it crash.
It is working with VPL on studio, but not on studio alone. 
I suggest a problem with the feedback.

**Comment by stephanemagnenat** [comment]
I confirm this occurs on Linux as well.

**Comment by stephanemagnenat** [comment]
Fixed in commit 20d9ba0, please try latest generated builds (maybe wait some minutes or check on the status page (http://lsropc28.epfl.ch:8080/view/monitor/) if you have EPFL VPN access).

**Comment by mbonani** [comment]
All is fine  for all version
every thing is onlie for user
